strongbox.go

```diff
@@ -138,6 +130,15 @@ func main() {
 		usage()
 	}
 
+	if *flagIdentityFile != "" {
```

Comment:
This if statement is redundant and can be removed, so always do `identityFilename = *flagIdentityFile` after `flag.Parse()`